lib/tasks/segment/segment.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_dependency 'csv'
+
+namespace :segment do
+  desc 'Save as deleted objects the users from the imported segment csv'
+  task :save_deleted_users, [:file_name] => [:environment] do |_task, args|
+    file_path =  Rails.root.join('config', args[:file_name])
```

Comment:
Why in `config`?

Reply:
Right, it is not configuration, it is actually the user data... but where then? 🤔

Reply:
I've just made it as "responsibility of the user to pass the whole path, be it in config or anywhere else"